docs/source/data/kedro_io.md

```diff
@@ -469,7 +469,7 @@ def create_partitions() -> Dict[str, Callable[[], Any]]:
 ```
 
 ```{note}
-When using lazy saving the dataset will be written _after_ the `after_node_run` [hook](../extend_kedro/hooks).
+When using lazy saving, the dataset will be written _after_ the `after_node_run` [hook](../hooks/hooks).
```

Comment:
I'm wondering if we should perhaps rename this section to `introduction.md` so that the URL looks like `hooks/introduction`. If this change is made then `hooks_examples` should be `examples`.